wordfreq/__init__.py

```diff
@@ -165,17 +84,13 @@ def tokenize(text, lang):
     first, so that they can be expected to match the data.
     """
     if lang == 'ja':
-        global mecab_tokenize
-        if mecab_tokenize is None:
-            from wordfreq.mecab import mecab_tokenize
+        from wordfreq.mecab import mecab_tokenize
```

Comment:
Hmm.  Isn't this called each time you want to look up a word?  And didn't we just get through the fact that `mecab_tokenize()` lacks the machinery that its lumi_science counterpart has to deal with the finite mecab buffer?